vars/withAcrClient.groovy

```diff
@@ -0,0 +1,30 @@
+import uk.gov.hmcts.contino.Environment
+
+def withRegistrySecrets(Closure block) {
+  def registrySecrets = [
+    [$class: 'AzureKeyVaultSecret', secretType: 'Secret', name: 'registry-name', version: '', envVariable: 'REGISTRY_NAME'],
```

Comment:
I would consider moving these out of keyvault, it doesn't really add any value.
if anything they should be env vars on jenkins

Reply:
As discussed, will leave this for AB#95 to handle.